test/e2e/network_test.go

```diff
@@ -299,8 +300,13 @@ func (tc *testContext) deleteNamespace(name string) error {
 
 // deployWindowsWebServer creates a deployment with a single Windows Server pod, listening on port 80
 func (tc *testContext) deployWindowsWebServer(name string, affinity *v1.Affinity) (*appsv1.Deployment, error) {
+	powerShellExe := "pwsh.exe"
```

Comment:
Will this change be required [here](https://github.com/openshift/windows-machine-config-operator/pull/321/files#diff-84675b994e150b2fe4128b3f0a171854d1060ba909b9ff0aeb8833c8d9f4dae9R491) as well?

Reply:
I don't follow. [here](https://github.com/openshift/windows-machine-config-operator/pull/321/files#diff-84675b994e150b2fe4128b3f0a171854d1060ba909b9ff0aeb8833c8d9f4dae9R491) takes me back this same line.

Reply:
I think you are asking about the `getWinCurlerCommand`. Yes, it does.